### ISSUE TITLE
does the mail stuff work?

### ISSUE THREAD

**Issue by chppppp** [briancappello/flask-react-spa#14]
I put my information for the mail section in config.py but i do not get any confirmation emails. Is this section working?

```
    ##########################################################################
    # mail                                                                   #
    ##########################################################################
    MAIL_ADMINS = ('myemail@gmail.com',)  # FIXME
    MAIL_SERVER = os.environ.get('FLASK_MAIL_HOST', 'smtp.gmail.com')
    MAIL_PORT = int(os.environ.get('FLASK_MAIL_PORT', 587))
    MAIL_USE_TLS = get_boolean_env('FLASK_MAIL_USE_TLS', True)
    MAIL_USE_SSL = get_boolean_env('FLASK_MAIL_USE_SSL', False)
    MAIL_USERNAME = os.environ.get('FLASK_MAIL_USERNAME', 'myemail@gmail.com')
    MAIL_PASSWORD = os.environ.get('FLASK_MAIL_PASSWORD', 'myapppassword')
    MAIL_DEFAULT_SENDER = (
        os.environ.get('FLASK_MAIL_DEFAULT_SENDER_NAME', 'Flask React SPA'),
        os.environ.get('FLASK_MAIL_DEFAULT_SENDER_EMAIL',
                       f"noreply@{os.environ.get('FLASK_DOMAIN', 'localhost')}")
    )
```

**Comment by GeeCastro** [comment]
Hi @chppppp. Try editing the [fixture.json](https://github.com/briancappello/flask-react-spa/blob/master/fixtures.json) file.

**Comment by chppppp** [comment]
Thanks for your help @Chichilele. `fixtures.json` appears to be used to pre-populate the database with a valid admin and user account. I went ahead and edited the admin account's email address directly through the database. I then tried registering an account and still am not receiving any mail. 

Just to be safe I edited the actual `email` field for the admin in fixtures.json and that still did not produce the expected results (receiving an email). 

Thanks
